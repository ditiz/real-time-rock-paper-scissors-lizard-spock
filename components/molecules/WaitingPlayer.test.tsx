import "@testing-library/jest-dom"
import { render, screen } from "@testing-library/react"
import WaitingPlayerMolecule from "./WaitingPlayer"

it("renders WaitingPlayerMolecule unchanged", () => {
  const { container } = render(<WaitingPlayerMolecule />)
  expect(container).toMatchSnapshot()
})

it("should contain waiting message", () => {
  render(<WaitingPlayerMolecule />)
  expect(screen.getByText(/waiting/i)).toBeTruthy()
})
