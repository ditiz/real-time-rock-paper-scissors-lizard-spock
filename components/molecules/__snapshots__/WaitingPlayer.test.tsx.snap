// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders WaitingPlayerMolecule unchanged 1`] = `
<div>
  <article
    class="game__waiting-message"
  >
    Waiting for another player
  </article>
</div>
`;
